src/containers/post/post.component.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { addPost } from "database/addPost";
+
+const Post = () => {
+  const addPostToDB = () => {
+    addPost(document.getElementById("post").value);
```

Comment:
Can we remove this logic and do an `onChange` on the `<input />` like we did earlier today?

Reply:
I changed this in the next PR